### ISSUE TITLE
meta-llama/Llama-3.2-1B, The model is not working properly

### ISSUE THREAD

**Issue by yangh0597** [pytorch/executorch#6848]
### 📚 The doc issue

I use this command transform model(Llama-3.2-1B)
```
python -m examples.models.llama.export_llama --checkpoint "${MODEL_DIR}/consolidated.00.pth" -p "${MODEL_DIR}/params.json" -kv --disable_dynamic_shape --qnn --pt2e_quantize qnn_16a16w -d fp32 --num_sharding 16 --metadata '{"get_bos_id":128000, "get_eos_ids":[128009, 128001]}' --output_name="test.pte"
```

but  the result is whis when i run it on phone
![image](https://github.com/user-attachments/assets/713b9b3f-2381-4d4c-8c42-09f3806b03de)

**Comment by cariad-v** [comment]
how did you manage to convert the model using the qnn backend? My Llama3.2-1B-Instruct model gets converted to .pte, but I cannot load it in the application, I get the error as mentioned here: #6820

**Comment by v-prgmr** [comment]
And second thing that comes to my mind now: Llama-3.2-1B-Instruct is what you will need for chat purposes, Llama-3.2-1B is the base model @yangh0597

**Comment by yangh0597** [comment]
> And second thing that comes to my mind now: Llama-3.2-1B-Instruct is what you will need for chat purposes, Llama-3.2-1B is the base model @yangh0597
I tried, but it didn't work

**Comment by v-prgmr** [comment]
Yeah so if you try converting the Llama-3.2-1B-Instruct model (both the vanilla models and SpinQuant models) to run on XNNPACK backend, you should be able to get it working without loss of accuracy. I can also confirm that the Llama-3.2-1B-Instruct model built for QUALCOMM backend throws errors just like yours. @yangh0597

**Comment by nil-is-all** [comment]
Hi @yangh0597, are you still facing this issue?
